patches/chrome-browser-ui-views-tabs-tab_hover_card_bubble_view.h.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/ui/views/tabs/tab_hover_card_bubble_view.h b/chrome/browser/ui/views/tabs/tab_hover_card_bubble_view.h
+index eaa40d8fdb646fe7ae39a2f363b3e8405519509b..4c8c87dbb55406de39718f3870f2c9cad311a253 100644
+--- a/chrome/browser/ui/views/tabs/tab_hover_card_bubble_view.h
++++ b/chrome/browser/ui/views/tabs/tab_hover_card_bubble_view.h
+@@ -62,6 +62,7 @@ class TabHoverCardBubbleView : public views::BubbleDialogDelegateView,
+   }
+ 
+   private:
++  BRAVE_TAB_HOVER_CARD_BUBBLE_VIEW_H_
```

Comment:
header overrides should always go at the end of `public`

Reply:
example https://github.com/brave/brave-core/pull/3620/files#diff-8bbc28fd6d32df2fc7997694b8be71b3R9 and https://github.com/brave/brave-core/pull/3620/files#diff-008587a3a9a0fa6cd4dda0377233ed4bR9

Reply:
Fixed.